app/models/purl.rb

```diff
@@ -20,6 +20,29 @@ class Purl < ActiveRecord::Base
   }
 
   ##
+  # Return true targets with always values only if the object is not deleted in
+  # purl mount
+  # @return [Array]
+  def computed_true_targets
```

Comment:
Yeah I'm not sure about that. I opted for the name `computed_true_targets` (not a great name I realize) so that someone wouldn't assume it just returned back whatever was there. What do you think?